### PR TITLE
chore(monorepo): add testing to pr template

### PR DESCRIPTION
**Pull request by alber70g**
To improve the quality and speed of the reviewers work, we've added a two sections to be filled when submitting a PR.

This allows us to be consious about what they're testing, and allows the reviewer to give feedback on what is tested. It also allows the reviewer to get a head start when working through the PR

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -8,3 +8,49 @@ Thanks for contributing to this project!
 - Run `pnpm install` and `pnpm test`
 - In short: help us help you to get this through!
```
Comment (Ghislain89) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Do we still want to keep first few lines in?

Comment (alber70g) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: Probably not. I'll remove it

**Review by Ghislain89 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,45 @@
...
+
+Related Issue/Asana ticket: \_
+
+Short description: \_
+
+<!--
+Examples:
+
+Title: Add Search Functionality to User Dashboard.
+Related Issue: #1234
```
Comment (lkononenko) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: Does Asana ticket have number? I didn't find it (except the id that is changing in the url)

Comment (alber70g) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: No it doesn't. My mistake. I've updated the examples

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,10 +1,45 @@
...
+* Stress Test: Executes the data processing module under high load to validate stability improvements.
 -->
+
+### Reminders (if applicable)
+
+- [ ] I ran `pnpm install` and `pnpm test` in the root of the monorepo
+      (optionally with `--filter=...package...` to exclude non-affected
+      projects)
+- [ ] I ran `pnpm changeset` in the root of the monorepo
+- [ ] Test coverage has not decreased
```
Comment (Ghislain89) [inline .github/PULL_REQUEST_TEMPLATE.md:45]:
```suggestion
- [ ] Test coverage has not decreased
- [ ] Docs have been updated to reflect changes in PR (don't forget docs.kadena.io)
```

**Pull request merged**